README.md

```diff
@@ -124,6 +124,9 @@ Even though we will gladly assist you in finishing up your PR, try to
 - check the output of `cargo clippy --all` or `./clippy.sh`
 - run `cargo build --no-default-features` to check for `no_std` compatibility (and possibly add `#[cfg(feature = "std")]`) to hide parts of your code.
 
+Some checks can be performed automatically during commit using [pre-commit](https://pre-commit.com/).
```

Comment:
Do we need thsi in the main readme? Maybe have a Development.md readme or book page?